test_runner/src/main/kotlin/ftl/util/LogTableBuilder.kt

```diff
@@ -12,16 +12,19 @@ data class TableColumn(
     val header: String,
     val data: List<String>,
     val dataColor: List<SystemOutColor> = listOf(),
-    val columnSize: Int = ((data + header).maxByOrNull { it.length }?.length ?: 0) + DEFAULT_COLUMN_PADDING
+    val columnSize: Int = ((data + header).maxByOrNull { it.length }?.length ?: 0) + DEFAULT_COLUMN_PADDING,
+    val align: Align
```

Comment:
As @piotradamczyk5 mentioned, Align.LEFT could set as default, also here.

Reply:
@jan-gogo @piotradamczyk5 
IMO it's clearer to force explicit alignment set up and I do not fully agree with you.
On the other hand I don't have very strong opinion about it and I'm in minority in that case, so sure, I'll change it 👍🏻